frog/imports/api/templates.js

```
@@ -0,0 +1,61 @@
+import { Meteor } from 'meteor/meteor';
+import { Mongo } from 'meteor/mongo';
+import { uuid } from '/imports/frog-utils';
+import { Graphs } from './graphs';
+
+export const Templates = new Mongo.Collection('templates');
+
+export const addTemplate = (name, graph) => {
+  const templateId = uuid();
+  Templates.insert({
+    name: name,
+    graph: graph,
+    _id: templateId,
+    ownerId: Meteor.userId(),
+    createdAt: new Date()
+  });
+};
+
+export const findTemplate = id => {
+  if (id) {
+    const template = Templates.findOne(id);
+    return template;
+  }
+  return null;
+};
+
+export const updateTemplate = (id, graph) => {
+  try {
```

Under which condition would this fail? Silently swallowing errors is dangerous (although I know I do it some places :))

---

Ah, this was when I was debugging. I forgot to revert back.